template-compose/app/build.gradle.kts

```diff
@@ -133,54 +134,25 @@ dependencies {
 
     implementation(fileTree(mapOf("dir" to "libs", "include" to listOf("*.jar"))))
 
-    with(Dependencies.AndroidX) {
-        implementation(CORE_KTX)
-        implementation(LIFECYCLE_RUNTIME_KTX)
-        implementation(LIFECYCLE_RUNTIME_COMPOSE)
-        implementation(DATASTORE_PREFERENCES)
```

Comment:
DATASTORE_PREFERENCES should not included into the basic bundle `androidx`. Ideally, we just combine basic dependencies to quickly add them to a module, not everything of a group 🙏 
```
androidx = [
    "androidx-core",
    "androidx-lifecycle-runtime",
    "androidx-lifecycle-compose",
]
```

Reply:
Agree :+1